samza-core/src/main/java/org/apache/samza/config/JobConfig.java

```diff
@@ -47,6 +47,17 @@
   public static final String CONFIG_REWRITERS = "job.config.rewriters";
   public static final String CONFIG_REWRITER_CLASS = "job.config.rewriter.%s.class";
 
+  /**
+   * job.config.loader.factory specifies {@link ConfigLoaderFactory} to get {@link ConfigLoader}
+   */
+  public static final String CONFIG_LOADER_FACTORY = "job.config.loader.factory";
+  /**
+   * job.config.loader.properties prefix wraps properties needed for {@link ConfigLoader} to loader config.
+   * e.g. {@link org.apache.samza.config.loaders.PropertiesConfigLoaderFactory}
+   * will read job.config.loader.properties.path to locate the config file location
+   */
+  public static final String CONFIG_LOADER_PROPERTIES_PREFIX = "job.config.loader.properties.";
```

Comment:
where is it used? if it used only in the Loader, should't it be defined there too?

Reply:
Yes, it is supposed to be only used by config loader, which specifies properties it needed to retrieve full job config. 

Are you suggesting we move this to ConfigLoader instead?

Reply:
yea, because if it is not used here - there is no point keeping it here.

Reply:
Moved to ConfigLoaderFactory